src/styles.css

```diff
@@ -0,0 +1,88 @@
+body {
+  background-color: #f4f2f3;
+}
+
+main {
+  display: flex;
+  align-items: center;
+  height: 95vh;
+  justify-content: center;
+}
+
+.to_do_list_container {
+  border: 1px solid #9497ae;
+  box-shadow: 0.05rem 0.05rem 2rem;
+  width: 40rem;
+  height: auto;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: auto;
+  padding: 1rem;
+  border-bottom: 1px solid #c4c6d8;
+  background-color: white;
+}
+
+h1 {
+  margin: 0;
+  font-family: Quintessential, sans-serif;
+  font-size: 1.8rem;
+  color: black;
+}
+
+.reload {
+  align-self: center;
+  color: #c4c6d8;
+  margin-right: 1rem;
+  cursor: pointer;
+}
+
+.task_input {
+  background-color: white;
+  border-bottom: 1px solid #c4c6d8;
+}
+
+.task_input input {
+  font-size: 1.5rem;
+  padding: 0.5rem;
+  font-family: Quintessential, sans-serif;
+  width: fit-content;
+  border: none;
+  outline: none;
+  margin-left: 1rem;
+}
+
+.task_container {
+  display: flex;
+
+  /* padding: 1rem 0; */
```

Comment:
- [ ] Same here. I would recommend removing the commented code to make the codebase cleaner.